src/lib/frontend/models.ml

```diff
@@ -245,221 +231,19 @@ module Pp_smtlib_term = struct
 
 end
 
-module SmtlibCounterExample = struct
-  let fresh_counter = ref 0
-
-  let reset_counter () = fresh_counter := 0
+(* of module SmtlibCounterExample *)
```

Comment:
Not sure what this means?

Reply:
I guess it's a rebase artefact. It was used to delimit a module that doesn't exist anymore.